vars/artifactSetVersion.groovy

```diff
@@ -84,7 +84,7 @@ def call(Map parameters = [:], Closure body = null) {
                 .withMandatoryProperty('gitSshUrl')
                 .use()
             
-            sh 'git add .'
+            sh 'git add --update .'
```

Comment:
could interfere with the [closure](https://github.com/SAP/jenkins-library/blob/master/vars/artifactSetVersion.groovy#L28)